axelor-budget/src/main/resources/views/GlobalBudget.xml

```diff
@@ -0,0 +1,383 @@
+    model="com.axelor.utils.db.Wizard">
+    <panel>
+      <field name="$budgetVersion" title="Budget Version" type="many-to-one"
+        target="com.axelor.apps.budget.db.BudgetVersion" colOffset="3" widget="SuggestBox"
+        grid-view="budget-version-grid" form-view="budget-version-form"
+        onSelect="action-global-budget-method-budget-version-domain"/>
+      <button name="validate-change-global-budget-version" title="Validate"
+        onClick="action-method-global-budget-budget-version-change,close"/>
+    </panel>
+  </form>
+
+
+  <action-view name="action-view-global-budget-budget-version-new"
+    title="Create New version" model="com.axelor.utils.db.Wizard">
+    <view type="form" name="global-budget-budget-version-new-wizard-form"/>
+    <view-param name="popup" value="reload"/>
+    <view-param name="popup-save" value="false"/>
+    <view-param name="show-toolbar" value="false"/>
+    <view-param name="show-confirm" value="false"/>
+    <context name="_globalBudget" expr="eval: __this__"/>
+  </action-view>
+
+  <action-view name="action-view-global-budget-budget-version-change"
+    title="Change budget version" model="com.axelor.utils.db.Wizard">
+    <view type="form" name="global-budget-budget-version-change-wizard-form"/>
+    <view-param name="popup" value="reload"/>
+    <view-param name="popup-save" value="false"/>
+    <view-param name="show-toolbar" value="false"/>
+    <view-param name="show-confirm" value="false"/>
+    <context name="_globalBudget" expr="eval: __this__"/>
+  </action-view>
+
+  <action-method name="action-method-global-budget-budget-version-change">
+    <call class="com.axelor.apps.budget.web.GlobalBudgetController"
+      method="changeBudgetVersion"/>
+  </action-method>
+
+  <action-method name="action-method-budget-version-new">
+    <call class="com.axelor.apps.budget.web.GlobalBudgetController"
+      method="createNewBudgetVersion"/>
+  </action-method>
+
+  <action-method name="action-method-budget-default-version-new">
+    <call class="com.axelor.apps.budget.web.GlobalBudgetController"
+      method="createDefaultBudgetVersion"/>
+  </action-method>
+
+  <action-method name="action-global-budget-method-budget-version-domain">
+    <call class="com.axelor.apps.budget.web.GlobalBudgetController"
+      method="setBudgetVersionDomain"/>
+  </action-method>
+
+  <action-view
+    name="action-global-budget-line-distribution-view-show-simulated-moves-lines"
+    title="Simulated Moves" model="com.axelor.apps.budget.db.BudgetDistribution">
+    <view type="grid" name="budget-distribution-simulated-moves"/>
+    <view type="form" name="budget-distribution-line-form"/>
+    <view-param name="show-toolbar" value="true"/>
+    <view-param name="show-confirm" value="true"/>
+    <domain>self.budget.budgetLevel.parentBudgetLevel.globalBudget.id= :_globalBudgetId and
+      self.moveLine.move.statusSelect = 5</domain>
```

Comment:
it's better to add CDATA in all domains, else the formatting might break the query